src/extension/manifest.json

```
@@ -0,0 +1,90 @@
+{
+    "manifest_version": 3,
+    "name": "__MSG_title__",
+    "version": "6.35.2",
```

Sync with `package.json`
```suggestion
    "version": "0.0.1",
```